code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -39,10 +39,8 @@
 
 /obj/item/weapon/storage/MouseDropFrom(obj/over_object as obj)
 	if(over_object == usr && (in_range(src, usr) || is_holder_of(usr, src)))
-		orient2hud(usr)
-		if(usr.s_active)
-			usr.s_active.close(usr)
```

Comment:
Closing of `s_active` is already being handled in `show_to`, this redundancy is being used throughout the codebase and should be removed where found.